packages/schematics/angular/library/index.ts

```diff
@@ -181,6 +181,10 @@ export default function (options: LibraryOptions): Rule {
         export: true,
         standalone: options.standalone,
         project: packageName,
+        // Explicitly set an empty `type` since it doesn't necessarily make sense in a library.
+        // This also ensures that the generated files are valid even if the `component` schematic
+        // inherits its `type` from the workspace.
+        type: '',
```

Comment:
I was on the fence between this and adding a `type` as an option. I decided not to add the option, because IMO it doesn't make sense in a library context, especially if it's a library that's being generated for the first time.